shared/rakefiles/xk_infra.rake

```diff
@@ -1,15 +1,16 @@
-common_sa_organization_permissions = [
+common_sa_organization_roles = [
   "roles/dns.admin",
   "roles/iam.organizationRoleViewer",
   "roles/iam.serviceAccountAdmin",
   "roles/iam.serviceAccountKeyAdmin",
   "roles/resourcemanager.projectIamAdmin",
   "roles/resourcemanager.projectCreator",
   "roles/serviceusage.serviceUsageAdmin",
-  "roles/storage.admin"
+  "roles/storage.admin",
+  "roles/viewer"
```

Comment:
Where do you find this information? [Documentation](https://cloud.google.com/iam/docs/understanding-roles) says nothing about viewer role deprecation...

Reply:
I think I was conflating "primitive" with "deprecated.

However, it is a best practice to avoid using primitive roles: https://forsetisecurity.org/docs/latest/concepts/best-practices.html

However however, this use case is a reasonable exception. I'd say the second point from this doc applies: https://cloud.google.com/iam/docs/faq#when_would_i_use_primitive_roles

In conclusion, I retract this suggestion.